### PR TITLE
Adjust PriceCalculator layout

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Adjust PriceCalculator layout 
- Make sure layout is visually centered in desktop and not expanding viewport height
- Adjust position in mobile
- Replace inline styles with classnames

### Before
Screenshot 2024-09-30 at 14 50 09

Screenshot 2024-09-30 at 14 49 58

### After
Screenshot 2024-09-30 at 12 08 47

Screenshot 2024-09-30 at 14 58 47

## Justify why they are needed
Match design
## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
* **#4806** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4806?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gustaveen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/features/priceCalculator/PriceCalculatorCmsPage.css.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { HEADER_HEIGHT_MOBILE } from '@/components/Header/HeaderMenuMobile/Heade
 export const pageGrid = style({
   display: 'grid',
   gridTemplateRows: 'min-content 1fr',
-  gap: tokens.space.md,
+  columnGap: tokens.space.md,
```
Comment (gustaveen) [inline apps/store/src/features/priceCalculator/PriceCalculatorCmsPage.css.ts:9]: Make sure we don't add row gap that makes the sections to tall

File: apps/store/src/features/priceCalculator/InsuranceDataForm/InsuranceDataForm.css.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  marginTop: '-1rem', // Offset from default gap of 2rem
+})
+
+export const formSection = styleVariants({
+  base: {
+    marginTop: '3.75rem',
+  },
+  ssn: {
+    ...responsiveStyles({
+      lg: {
```
Comment (alebedev) [inline apps/store/src/features/priceCalculator/InsuranceDataForm/InsuranceDataForm.css.ts:15]: Centering with absolute, transform and negative margins feels hacky. Can we do it with flex or grid?

File: apps/store/src/features/priceCalculator/InsuranceDataForm/InsuranceDataForm.tsx [hunk 3]
```diff
@@ -84,7 +80,10 @@ export function InsuranceDataForm() {
       )
     }
     return (
-      <div key={section.id} className={yStack({ gap: 'lg' })} style={sectionStyle}>
+      <div
+        key={section.id}
+        className={clsx(formSection.base, isSsnSection && formSection.ssn, yStack({ gap: 'lg' }))}
```
Comment (alebedev) [inline apps/store/src/features/priceCalculator/InsuranceDataForm/InsuranceDataForm.tsx:85]: Should `yStack` come last for priority reasons?  It's usually first

**Review by alebedev — APPROVED** [review]
Looks good! Few comments

File: apps/store/src/features/priceCalculator/InsuranceDataForm/InsuranceDataForm.tsx (hunk 3 above)
Comment (gustaveen) [reply apps/store/src/features/priceCalculator/InsuranceDataForm/InsuranceDataForm.tsx:85]: Shouldn't overlap but yeah that might be a good practice 👍

File: apps/store/src/features/priceCalculator/InsuranceDataForm/InsuranceDataForm.css.ts (hunk 2 above)
Comment (gustaveen) [reply apps/store/src/features/priceCalculator/InsuranceDataForm/InsuranceDataForm.css.ts:15]: We should use flex or grid for sure where possible. This is a bit of a edge case though where it's only one section that should be positioned vertically centered and it's nested in several containers that we would need to modify. But I'll see if it's possible to simplify 👍 @gustaveen

**Pull request closed without merging**